misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/LocalDateTimeFeatureAssertionsSpec.kt

```diff
@@ -23,7 +23,9 @@ import ch.tutteli.atrium.translations.DescriptionDateTimeLikeAssertion
 import org.spekframework.spek2.Spek
 import org.spekframework.spek2.style.specification.Suite
 import java.time.DayOfWeek
+import java.time.LocalDate
```

Comment:
remove unused import

and basically same input as for LocalDateFeatureAssertionsSpec